helm/cert-manager-app/values.yaml

```diff
@@ -10,3 +10,10 @@ global:
 image:
   name: giantswarm/cert-manager-controller
   tag: v0.9.0
+
+resources:
+  requests:
+    cpu: 50m
+    memory: 100Mi
+  limits:
+    memory: 200Mi
```

Comment:
I think it was temp spike during startup of cert-manager that was making it go over requests. minute or so after startup it went below requests like on other clusters 

So maybe we should remove limit, but add alert if requests is too much over the requests or too close to requests for long enough. With requests != limits it's anyway burstable, not guaranteed. Without limits it would be more flexible, tolerate better these kind of short spikes. And it's really hard to set the limit well, without measuring and understanding app and all the factors that could be contributing to the spike in its memory usage.